src/main/java/com/checkout/AbstractCheckoutSdkBuilder.java

```diff
@@ -21,11 +19,19 @@
         return this;
     }
 
+    /**
+     * @deprecated in favor to use Environment
+     */
+    @Deprecated
```

Comment:
Can you please provide a more informative message here? I was thinking of something like: "In a future release, setting a custom URI won't be allowed as it will use the same environment URI set in #environment()"